Dockerfile

```
@@ -1,6 +1,6 @@
 FROM debian:buster as dump1090
 
-ENV DUMP1090_VERSION v3.8.1
+ENV DUMP1090_VERSION v4.0
```

```suggestion
ENV DUMP1090_VERSION v5.0
```
Version 5.0 is tagged now https://github.com/flightaware/dump1090/commit/f862b530c8cea51f4bb9e72356579be67818dcaf